lib/Controller/ImageController.php

```diff
@@ -377,14 +378,75 @@ private function getImageJPEG($blob, $mimetype): array
         } catch (\ImagickException $e) {
             throw Exceptions::Forbidden('Imagick failed to read image: '.$e->getMessage());
         }
-
         // Convert to JPEG
         try {
             $image->autoOrient();
-            $image->setImageFormat('jpeg');
-            $image->setImageCompressionQuality(95);
+            $format = $this->config->getSystemValueString('memories.highres_format', 'jpeg');
```

Comment:
`Util::getSystemConfig`. Also see `Util::systemConfigDefaults`.

The config values need to be correctly scoped, e.g. `memories.image.highres.format`